### PR TITLE
bump cardano-node's libraries to 1.9.1

### PR DESCRIPTION
**Pull request by KtorZ**
# Issue Number

N/A

# Overview

- 8c0e6eb005ada34d261bc89ba36a80e803b487a7
  :round_pushpin: **bump cardano-node's libraries to 1.9.1**
  This should fix the Windows build issues on Hydra

# Comments

### CODE REVIEW

**Review by rvl — APPROVED** [review]
Looks fine

File: stack.yaml [hunk 1]
```diff
@@ -74,28 +74,28 @@ extra-deps:
 - gray-code-0.3.1
 
 - git: https://github.com/well-typed/cborg
-  commit: 42a83192749774268337258f4f94c97584b80ca6
+  commit: 8e12924f5b34ac9727a481ef455fc5629f42be8c
```
Comment (dcoutts) [inline stack.yaml]: This is going backwards by 2 commits. Was this intended?

**Review by dcoutts — APPROVED** [review]
LGTM. Just one query below.

File: stack.yaml (hunk 1 above)
Comment (KtorZ) [reply stack.yaml]: Hmm. No it isn't. We do want the pretty print fix.

**Pull request merged**